Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -370,7 +428,9 @@ public class FeaturevisorInstance {
                     enabled: stickyFeature.enabled,
                     sticky: stickyFeature)
 
-                  logger.debug("using sticky enabled", ["featureKey": featureKey]) // TODO: Log evaluation object. Make it encodable
+            let evaluationDictionary = evaluation.toDictionary(evaluation)
+
+            logger.debug("using sticky enabled", evaluationDictionary)
```

Comment:
Skip creating a variable as this is only for logging and invoke this as:
`logger.debug("using sticky enabled", evaluation.toDictionary())`